sources/host-ctr/cmd/host-ctr/all_devices_allowed.go

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package main
+
+import (
+	"context"
+
+	"github.com/containerd/containerd/containers"
+	"github.com/containerd/containerd/oci"
+	runtimespec "github.com/opencontainers/runtime-spec/specs-go"
+)
+
+// withAllDevicesAllowed permits all access on all devices nodes for the container
+// Taken from https://github.com/containerd/containerd/blob/25947db049b058fcbce291ef883b8b512e3ea440/oci/spec_opts.go#L1010
+// which is not available in containerd v1.3.7
+// TODO Once we move to containerd v1.4.0 remove this and use oci.WithAllDevicesAllowed
```

Comment:
Can this become an issue, and we can reference the issue ID here?

Reply:
https://github.com/bottlerocket-os/bottlerocket/issues/1038